corehq/apps/users/migrations/0060_grant_web_apps_default_mobile_worker.py

```diff
@@ -0,0 +1,25 @@
+# Gives the default mobile worker role access to web apps
+from django.db import migrations
+
+from corehq.apps.users.models_role import UserRole
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _update_default_mobile_worker_role(apps, schema_editor):
```

Comment:
I'm also a bit nervous about the access change associated with this.  There's always the possibility that there are mobile workers using roles other than the default _and_ using web apps.  I do think the current behavior is arguably a bug (if the role explicitly doesn't have web apps access...), so I think we have some wiggle room.  Still, is there anything we could do to de-risk this, or get a sense of how many users it might realistically impact?

Reply:
Point well taken. I'll do some digging on this and report back.

(You reviewed this a lot faster than I was expecting - thank you!)

Reply:
reverted this change, see longer comment below